ckanext/followy/tests/e2e/followy_test.py

```diff
@@ -41,11 +41,12 @@ def test_create_dataset_and_follow(self):
         visibility != None
         next_page = self.browser.find_elements_by_class_name("btn-primary")[0]
         assert next_page != None
+        time.sleep(5)
```

Comment:
Is there any specific reason for choosing 5 secs and not any other number

Reply:
When running the test, It happens so fast that sometimes the page is not rendered properly before an assertion is checked thereby making the test erratic; adding a 5sec pause slows down the process a bit and allow pages to render completely thereby allowing elements of a page to be rendered fully and also the pause slows down the test so you get to see and understand what the test is doing(E2E test execute quite fast). There is no particular reason for choosing 5sec, I think is not too much time to take a pause while running the test.